docs/book/src/faq.md

```diff
@@ -92,9 +92,38 @@ securityContext:
 ```
 However, note that this problem is fixed and will not occur if you deploy the project in high versions (maybe >= 1.22).
 
+## The error `Too long: must have at most 262144 bytes` is faced when I run `make install` to apply the CRD manifests . How to solve it? Why this error is faced?
+
+The Kubernetes API has a size limit. Then, when the apply is used the API will add the annotation last-applied-configuration with all previous configuration which is exceeding this amount. [More info][k8s-obj-creation]
+
+Therefore, you have a few options to workround this scenario such as:
+
+- You can use the option `maxDescLen=0` to remove the description of your CRDs. Following an example:
+
+```shell
+
+ .PHONY: manifests 
+ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects. 
+     # Note that the option maxDescLen=0 was added in the default scaffold in order to sort out the issue 
+     # Too long: must have at most 262144 bytes. By using kubectl apply to create / update resources an annotation 
+     # is created by K8s API to store the latest version of the resource ( kubectl.kubernetes.io/last-applied-configuration). 
+     # However, it has a size limit and if the CRD is too big with so many long descriptions as this one it will cause the failure. 
+ 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases 
+
+You can review the design of your APIs and see if it has not more specs than should be by hurting single responsibility principle for example. So that you might to re-design them.
+
+Instead of Kubectl create command use Kubectl apply and it will work because kubectl create will send the request at once and it will fail while kubect apply send the request in small chunks so the desc len will be divided. 
```

Comment:
```suggestion
```

Remove this one. It seems there seems to be some confusion in that statement. The primary distinction between kubectl create and kubectl apply is not in how they send data (i.e., in chunks versus all at once), but in their approach to handling resources:

- kubectl create: This is used to create a resource. It requires that the resource does not already exist. If it does, the command will fail.
- kubectl apply: This manages applications through files defining Kubernetes resources. It creates and updates resources in a cluster through the file.

However, both commands send the configuration to the Kubernetes API in a similar manner. The statement about kubectl apply breaking the data into smaller segments is not accurate as per the documentation and general understanding.

The real difference in terms of the error you're referencing might come from how kubectl apply manages the last-applied-configuration annotation, but it doesn't break the request into smaller segments.

You could suggest to use. create instead of apply, however, that means also lost data. So we should not add it here.